### PR TITLE
Table and links

### PR DESCRIPTION
**Pull request by federicomarini**
Drafting the implementation of the final look for table and side content

### CODE REVIEW

File: R/shiny_fuse.R [hunk 1]
```diff
@@ -330,6 +324,32 @@ shiny_fuse <- function(out_annofuse = NULL) {
...
+        return(NULL)
+      } else {
+        tagList(
+          menuItem(
+            "Plot export settings", 
+            icon = icon("images"),
+            startExpanded = TRUE,
+            numericInput(
+              inputId = "export_width",
+              label = "Export width (in)",
```
Comment (jharenza) [inline R/shiny_fuse.R]: Hahah - accommodating the US folks using inches 😉 . 
```
              label = "Export width (in)",
```

**Review by jharenza — CHANGES_REQUESTED** [review]
Not sure if this is an issue in here or the general plotting function, but I am not seeing the `KIAA1549--BRAF` fusions load but see them loading as individual genes. Default plot settings. 

Screen Shot 2020-08-28 at 8 38 16 PM

Screen Shot 2020-08-28 at 8 44 42 PM

File: R/shiny_fuse.R (hunk 1 above)
Comment (federicomarini) [reply R/shiny_fuse.R]: :D no clue how widely used are cm on that side of the big pond 😛

**Comment by federicomarini** [comment]
I'd say it is more intrinsic to the plotting function. Looks like an NA is sneaking in?

Can you reproduce the plot also offline, not in shiny?

**Comment by kgaonkar6** [comment]
yeah it's the plotting function, checking

**Comment by kgaonkar6** [comment]
Screen Shot 2020-08-29 at 12 37 53 PM

there was a typo which counts the limits, seems to have fixed now

**Comment by federicomarini** [comment]
Looks like this branch is fully ready for a complete PR 😉

**Review by kgaonkar6 — APPROVED** [review]

**Review by jharenza — APPROVED** [review]
looks good!

**Comment by federicomarini** [comment]
Merging in with CLI, got a couple of conflicts that seem not possible to solve via web interface 😉

**Pull request merged**